sdk/containerregistry/azure-containers-containerregistry/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.6 (2022-1-10)
+## 1.0.0 (2022-01-11)
```

Comment:
Can't leave a comment on the right line, dependency updates have the wrong versions

Reply:
Shoot - I missed that. Thank you for catching it.